### PR TITLE
[#191] Checkout: header, filled info styles, tooltips, edit buttons

### PR DESCRIPTION
**Pull request by Aekal**
#191 
I added few new features to checkout according to the design:
- Different top header for checkout with the possibility of login and back to homepage
- Vertical line across circles
- New edit info buttons
- Tooltip component, which can by used by typing `<tooltip>Some text</tooltip>`

Header:

zrzut ekranu 2017-12-13 o 10 29 03

Filled info:

zrzut ekranu 2017-12-13 o 10 28 43

Tooltip:

zrzut ekranu 2017-12-13 o 10 43 05

### CODE REVIEW

File: src/themes/default/pages/Checkout.vue [hunk 1]
```diff
@@ -310,14 +310,21 @@ export default {
...
+    &:after {
+      content: '';
+      display: block;
+      position: absolute;
+      top: 0;
+      left: 17px;
+      z-index: -1;
+      width: 1px;
+      height: 100%;
+      background-color: #f2f2f2;
```
Comment (filrak) [inline src/themes/default/pages/Checkout.vue]: you should import the color variables here from `global_vars.scss`

File: src/themes/default/components/core/blocks/Checkout/Tooltip.vue [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  text-align: center;
+
+  &:before {
+    content: '';
+    display: block;
+    width: 0;
+    height: 0;
+    position: absolute;
+    border: 10px solid transparent;
+    border-top: 10px solid #333333;
```
Comment (filrak) [inline src/themes/default/components/core/blocks/Checkout/Tooltip.vue]: please import colors from `global_vars.scss`

File: src/themes/default/components/core/blocks/Checkout/Tooltip.vue [hunk 3]
```diff
@@ -0,0 +1,45 @@
+<template>
```
Comment (filrak) [inline src/themes/default/components/core/blocks/Checkout/Tooltip.vue]: shouldn't the tooltip be a global component?

**Comment by pkarw** [comment]
@Aekal could you fix it Today please?

**Comment by Aekal** [comment]
I'll do it in the evening

File: src/themes/default/components/core/blocks/Checkout/Tooltip.vue (hunk 3 above)
Comment (Aekal) [reply src/themes/default/components/core/blocks/Checkout/Tooltip.vue]: I moved tooltip to `components/core/Tooltip.vue` but I'm not sure if this is what you expect. 
Or should I use [globally registered component](https://vuejs.org/v2/guide/components.html#Global-Registration) instead?

File: src/themes/default/components/core/blocks/Checkout/Tooltip.vue (hunk 2 above)
Comment (Aekal) [reply src/themes/default/components/core/blocks/Checkout/Tooltip.vue]: Done

File: src/themes/default/pages/Checkout.vue (hunk 1 above)
Comment (Aekal) [reply src/themes/default/pages/Checkout.vue]: Done

**Comment by filrak** [comment]
thanks!

**Pull request merged**